test/extensions/filters/network/thrift_proxy/driver/generate_fixture.sh

```diff
@@ -83,26 +90,20 @@ else
 fi
 
 # start server
-"${DRIVER_DIR}/server" "${SERVICE_FLAGS[@]}" &
+"${DRIVER_DIR}/server.py" "${SERVICE_FLAGS[@]}" &
 SERVER_PID="$!"
 
 trap 'kill ${SERVER_PID}' EXIT;
 
-while [[ ! -a "${SOCKET}" ]]; do
-    sleep 0.1
-
-    if ! kill -0 "${SERVER_PID}"; then
-        echo "server failed to start"
-        exit 1
-    fi
-done
 
 if [[ -n "$HEADERS" ]]; then
     SERVICE_FLAGS+=("--headers")
     SERVICE_FLAGS+=("$HEADERS")
 fi
 
-"${DRIVER_DIR}/client" "${SERVICE_FLAGS[@]}" \
```

Comment:
@davinci26 I would guess this extension addition was required b/c of the PATHEXT variable not containing .py, seems like its still good to include unless we wanted to modify PATHEXT

Reply:
Turns out this one did not refer to the python file but rather to the executable file which is generated by bazel as part of `py_binary`